test/parallel/test-http-1.0.js

```diff
@@ -156,8 +156,8 @@ function test(handler, request_generator, response_validator) {
                               '\r\n';
 
     assert.strictEqual(expected_response, server_response);
```

Comment:
The order needs to be changed here too as expected_response comes after server_response